app/assets/javascripts/classes/MetricNodeVM.ts

```diff
@@ -41,6 +45,10 @@ class MetricNodeVM implements BrowseNode {
     expandMe() {
         this.expanded(this.expanded() == false);
     }
+
+    tooltip = ko.pureComputed(function() {
```

Comment:
@BrandonArp can you verify this is a decent way to add this method? I can't wrap my brain around mixing the typescript syntax with a `ko.pureComputed` instance